libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Support/CSFParameterStorage.m

```diff
@@ -63,6 +63,9 @@ - (BOOL)isEqual:(CSFParameterStorage*)object {
     if (![self.mimetypes isEqualToDictionary:object.mimetypes])
         return NO;
 
+    if (![self.bodyStreamBlock isEqual:object.bodyStreamBlock])
```

Comment:
Generally speaking, this `isEqual:` implementation doesn't cover the `nil == nil` case, since a `nil` parameter is always going to return `NO` for any comparison.